doc/content/integrations/cloud-integrations/daizy/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Daizy"
+description: ""
+weight:
+---
+
+Use [Daizy](https://daizy.io) to rapidly build, deploy and scale Internet of Things applications and stay in control of your data.
+
+The Daizy platform provides an intuitive management system for IoT devices and projects, and delivers data directly to any existing or future business systems. Daizy enables rapid scaling by reducing time to market for any IoT implementation while minimizing cost, improving flexibility and maintaining security.
+
+<!--more-->
+
+## Prerequisites
+
+1. A user account and organisation created within the [Daizy platform](https://portal.daizy.io/login).
+2. LoRa devices enrolled and deployed in your project.
```

Comment:
This is redundant, we assume devices are already sending data to TTS for all integrations

Reply:
The intent here was that the user needs the devices enrolled in their Daizy project, not in TTS, I will make this clear.